clicache/src/impl/PdxLocalReader.hpp

```diff
@@ -105,19 +105,19 @@ namespace Apache
         /// Read a 16-bit unsigned integer from the stream.
         /// </summary>
         /// <param name="fieldName">The name of a member field whose value to read.</param>
-        virtual System::UInt16 ReadUInt16( String^ fieldName );
+        virtual System::UInt16 ReadUShort( String^ fieldName );
 
         /// <summary>
         /// Read a 32-bit unsigned integer from the stream.
         /// </summary>
         /// <param name="fieldName">The name of a member field whose value to read.</param>
-        virtual System::UInt32 ReadUInt32( String^ fieldName );
+        virtual System::UInt32 ReadUInt( String^ fieldName );
         
         /// <summary>
         /// Read a 64-bit unsigned integer from the stream.
         /// </summary>
         /// <param name="fieldName">The name of a member field whose value to read.</param>
-        virtual System::UInt64 ReadUInt64( String^ fieldName );
+        virtual System::UInt64 ReadULong( String^ fieldName );
```

Comment:
A bit of a religious war here. I was in your camp originally. Using the bitlength suffix avoids any confusion. However, nearly all Microsoft docs use the C# keywords (short, int, long, and their unsigned counterparts (ushor, uint, ulong) instead of the .NET types (Int16, Int32, Int64, etc.). In light of this discovery I switched back to using the C# keywords in the API. I'm good either way, whatever the community prefers.